ivy/container/base.py

```diff
@@ -3989,6 +3992,58 @@ def __setitem__(self, query, val):
             New container after updating.
 
         """
+
+        def _map_fn(fn,x):
+            x = x.data if isinstance(x, ivy.Array) else x
+            return fn(x)
+
+        if query == "_backend":
+            self._backend = val
+            return
+
+        if query == "dynamic_backend":
+            from ivy.functional.ivy.gradients import _variable
+            from ivy.backend_handler import _determine_backend_from_args
+
+            if val == False:
+                self._backend = _determine_backend_from_args(self)
```

Comment:
Using `determine_backend_from_args` to determine the backend for the instance. This backend is then used to invoke the backend functions directly (i.e bypassing Ivy's API). 
Need to do this because I cant really use Ivy functions due to the `global dict` issue

Reply:
A similar suggestion to the local imports with the `ivy.Array` class